wiki/osu!api/pl.md

```diff
@@ -1,15 +1,18 @@
 ---
-outdated: true
-outdated_since: f90e775380f1c7041a62f3d7f5e3e29ba4326dee
+no_native_review: true
 ---
 
 # osu!api
 
 osu! posiada publiczne [API](https://en.wikipedia.org/wiki/API), którego używać mogą zewnętrzne usługi. W przeszłości miały do niego dostęp tylko upoważnione osoby, ale obecnie każdy użytkownik osu! może z niego korzystać.
 
-Dokumentacja API znajduje się na [oficjalnej wiki osu-api](https://github.com/ppy/osu-api/wiki). Obecna wersja API korzysta z prywatnych kluczy do autoryzacji żądań. Możesz ubiegać się o klucz API na [stronie rejestracji aplikacji](https://osu.ppy.sh/p/api) osu!.
+Dokumentacja API znajduje się na [oficjalnej wiki osu-api](https://github.com/ppy/osu-api/wiki). Obecna wersja API korzysta z prywatnych kluczy do autoryzacji żądań. Możesz ubiegać się o klucz API na [stronie rejestracji aplikacji](https://osu.ppy.sh/p/api) osu!. Miej na uwadze, że dokumentacja zawarta na wiki osu!api nie jest w pełni kompletna czy wyczerpująca, a samo API nie jest obecnie rozwijane. 
 
-Miej na uwadze, że dokumentacja zawarta na wiki osu!api nie jest w pełni kompletna czy wyczerpująca, a samo API nie jest obecnie rozwijane. Nowa wersja API jest w trakcie powstawania na [repozytorium osu-web](https://github.com/ppy/osu-web).
+---
+
+Nowa wersja API jest w trakcie powstawania na [repozytorium osu-web](https://github.com/ppy/osu-web) i celuje w bycie bardziej stabilną <!--I'm not sure how to translate the word "robust"--> niż oryginał. Dokumentacja (nie zawsze aktualna) może być znaleziona na jej [stronie tytułowej](https://docs.ppy.sh).
```

Comment:
The wording seems fine (can't see a better alternative for `robust`), but the sentence structure seems a bit anglicised. How about something like

```suggestion
Nowa wersja API jest w trakcie powstawania na [repozytorium osu-web](https://github.com/ppy/osu-web), zaś jej celem jest zwiększona stabilność względem oryginału. Dokumentację (niekoniecznie aktualną) można znaleźć na jej [stronie tytułowej](https://docs.ppy.sh).
```